rbi/stdlib/pathname.rbi

```diff
@@ -1060,11 +1060,17 @@ class Pathname < Object
     params(
         length: Integer,
         offset: Integer,
-        open_args: Integer,
+        external_encoding: T.any(String, Encoding),
+        internal_encoding: T.any(String, Encoding),
+        encoding: T.any(String, Encoding),
+        textmode: BasicObject,
+        binmode: BasicObject,
+        autoclose: BasicObject,
+        mode: String,
     )
     .returns(String)
   end
-  def read(length=T.unsafe(nil), offset=T.unsafe(nil), open_args=T.unsafe(nil)); end
+  def self.read(length=T.unsafe(nil), offset=T.unsafe(nil), external_encoding: T.unsafe(nil), internal_encoding: T.unsafe(nil), encoding: T.unsafe(nil), textmode: T.unsafe(nil), binmode: T.unsafe(nil), autoclose: T.unsafe(nil), mode: T.unsafe(nil)); end
```

Comment:
With this change, you've delete the sig for `Pathname#read` (a method on the instance) and replaced it with one for `Pathname.read` (a method the class itself). Was that intentional?

With this new version all the places that uses the method on an instance of `Pathname` now have an error reported on them.

Reply:
@neilparikh Apologies, typo on my end: fixed now.